packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -1197,24 +1197,23 @@ For more information about configuring an app for OpenID Connect, including grou
 
-> Native applications shouldn't provide -- and by default don't store -- the `client_secret` (see [Section 5.3.1 of the OAuth 2.0 spec](https://tools.ietf.org/html/rfc6819#section-5.3.1)). They can omit the `client_secret` from the request parameters when introspecting a token.
+Some endpoints require client authentication. To make requests to these endpoints, you must include a header or parameter in the request depending on the authentication method the application is configured with.
 
-Okta supports the following authentication methods:
+When registering an OAuth 2.0 client application, specify an authentication method by including the [token_endpoint_auth_method](https://developer.okta.com/docs/api/resources/apps/#add-oauth-2-0-client-application) parameter.
 
-> Note: You can use only one of these methods in a single request or an error occurs.
+> Note: If you don't specify a method when registering your client, the default method is `client_secret_basic`.
 
-* `client_secret_basic`, `client_secret_post`, `client_secret_jwt`: Use one of these methods to authenticate against any endpoint that requires client authentication, when the client has access to the `client_secret`.
+Okta supports the following authentication methods, detailed in the sections below:
 
-* `private_key_jwt`: Use this method when you want to use public/private key pairs for more security. The main benefit of this method is you can generate the private key on your own servers and never have it leave there for any reason, since you only need to provide the public key to Okta. This is better than `client_secret_jwt` since Okta must know what the `client_secret` string is beforehand, so there are more places that it could in theory be compromised. 
+* `client_secret_basic`, `client_secret_post`, `client_secret_jwt`: Use one of these methods when the client has a client secret. Public clients (such as single-page and mobile apps) that cannot protect a client secret must use `none` below.
 
-* `none` - Use this method when the client doesn't authenticate itself to the `/token` endpoint because it uses the [Implicit Flow](/authentication-guide/implementing-authentication/implicit/) or because it is a public client with no client secret or other authentication mechanism.
+* `private_key_jwt`: Use this when you want maximum security. This method is more complex and requires a server, so it also cannot be used with public clients.
```

Comment:
Try to make it more conversational in tone by using common contractions:

 `private_key_jwt`: Use this when you want maximum security. This method is more complex and requires a server, so it also can't be used with public clients.